src/sensors/SensorManager.cpp

```diff
@@ -108,6 +109,12 @@ namespace SlimeVR
             case IMU_ICM42688:
                 sensor = new ICM42688Sensor(sensorID, address, rotation, sclPin, sdaPin);
                 break;
+            case IMU_LSM6DSV:
+                {
+                uint8_t intPin = extraParam;
+                sensor = new LSM6DSVSensor(sensorID, imuType, address, rotation, sclPin, sdaPin, intPin);
+                }
+                break;
```

Comment:
```suggestion
                sensor = new LSM6DSVSensor(sensorID, imuType, address, rotation, sclPin, sdaPin, extraParam);
                break;
```

Reply:
I would suggest against this change, as the original change follow the pattern of how other cases in this switch statement has done this

![image](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/assets/32248439/4487c0fd-58af-47c5-bc10-cd8907f7960d)